torch/csrc/distributed/c10d/reducer.cpp

```diff
@@ -489,8 +497,8 @@ void Reducer::prepare_for_backward(
   std::vector<torch::autograd::Function*> queue;
```

Comment:
comments above `prepare_for_backward` also needs to be updated, as it is no longer marked as ready immediately.

Reply:
Thanks for noticing. Updated the comment.